  Source.cpp
d:\шаг\porga\ооп\dz\dz20\exceptions\exceptions\source.cpp(63): warning C4244: return: преобразование "__int64" в "int", возможна потеря данных
d:\шаг\porga\ооп\dz\dz20\exceptions\exceptions\source.cpp(60): error C4700: использована неинициализированная локальная переменная "number"
